### PR TITLE
Restore WASM debug hook (#7843)

### PR DESCRIPTION
**Pull request by ryzngard**

### CODE REVIEW

File: src/razor/src/extension.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ import { RazorFormatNewFileHandler } from './formatNewFile/razorFormatNewFileHan
 import { InlayHintHandler } from './inlayHint/inlayHintHandler';
 import { InlayHintResolveHandler } from './inlayHint/inlayHintResolveHandler';
 import { getComponentPaths } from '../../lsptoolshost/builtInComponents';
+import { BlazorDebugConfigurationProvider } from './blazorDebug/blazorDebugConfigurationProvider';
```
Comment (phil-allen-msft) [inline src/razor/src/extension.ts:56]: This looks like a copy of line 9.

Comment (phil-allen-msft) [reply src/razor/src/extension.ts:56]: @dibarbet, perhaps we don't need this particular cherry-pick.

Comment (dibarbet) [reply src/razor/src/extension.ts:56]: @ryzngard @phil-allen-msft I'm not seeing the fix in the prerelease branch, so as far as I know we need it
https://github.com/dotnet/vscode-csharp/blob/prerelease/src/razor/src/extension.ts

Comment (ryzngard) [reply src/razor/src/extension.ts:56]: I'll find out why the cherry pick didn't work

**Review by dibarbet — APPROVED** [review]

**Pull request merged**